init.d/service.fedora

```diff
@@ -6,9 +6,12 @@ ConditionVirtualization=!container
 Before=sysinit.target shutdown.target systemd-journald.service
 
 [Service]
-ExecStart=/usr/sbin/haveged -w 1024 -v 1 --Foreground
+ExecStartPre=+/usr/bin/sysctl -w kernel.random.write_wakeup_threshold=1024
```

Comment:
Shouldn't this be in `sysctl.d`?

Reply:
Why `sysctl.d`? The executable is `/usr/bin/sysctl`.

Or do you want to drop a file to `/usr/lib/sysctl.d/`? That would alter the value if the package is installed but the service is not started.